### ISSUE TITLE
net_certificate crashes when the domain does not have an SSL certificate

### ISSUE THREAD

**Issue by massyn** [turbot/steampipe-plugin-net#41]
**Describe the bug**
I am running `net_certificate` to identify domains with SSL issues.  If a domain does not have an SSL certificate, the plugin simply fails, and does not process anything.

```
Warning: executeQueries: query 1 of 1 failed: ERROR: rpc error: code = Unknown desc = TLS connection failed: dial tcp 164.92.154.37:443: i/o timeout (SQLSTATE HV000)
```

**Steampipe version (`steampipe -v`)**
`steampipe version 0.15.3`

**Plugin version (`steampipe plugin list`)**
`| hub.steampipe.io/plugins/turbot/net@latest         | 0.6.0   | net         |`

**To reproduce**
```
with domains as (
    SELECT 'steampipe.io' as domain
    UNION ALL SELECT 'massyn.com' as domain
)

SELECT
    N.domain,
    N.revoked
FROM
  net_certificate N
where
  N.domain IN (SELECT * FROM domains)
```

**Expected behavior**
In this example, I would expect the domain `massyn.com` to come up with a `null` value, instead of failing the script with the error message.

**Additional context**
None

**Comment by Subhajit97** [comment]
Thanks, @massyn, for reporting the issue.
We are currently working on this.  We will provide an update when it is available.

**Comment by Subhajit97** [comment]
Hi @massyn, sorry for the late response. We have pushed a fix for the above issue, and the changes will be available in the upcoming release. We will provide an update when it is available.

**Comment by Subhajit97** [comment]
Hey @massyn, this bug should now be fixed in the new `Net` plugin release [v0.7.0](https://github.com/turbot/steampipe-plugin-net/blob/main/CHANGELOG.md#v070-2022-08-17). If you're still seeing this error after updating, please comment and re-open this issue, thanks!
